src/components/Tables/DataTable/fragments/inputs/TableInputNumber.vue

```diff
@@ -0,0 +1,28 @@
+<template>
+  <InputNumber
+    :prefix="preset ? presets[preset].prefix : undefined"
+    :suffix="preset ? presets[preset].suffix : undefined"
+    :min-fraction-digits="
+      preset ? presets[preset].minFractionDigits : undefined
+    "
+    :format="!!preset"
+    :readonly="!edit"
+    :input-class="edit ? '' : '!border-none !shadow-none !bg-transparent'"
+  />
+</template>
+
+<script setup lang="ts">
+import InputNumber, { type InputNumberProps } from "primevue/inputnumber";
+
+export type TableInputNumberProps = {
```

Comment:
```suggestion
type TableInputNumberProps = {
```

se precisar do tipo em outro contexto, exporte-o em `types` e importe-o aqui